ProjectLighthouse.Servers.Website/Pages/Moderation/NewCasePage.cshtml.cs

```diff
@@ -43,14 +53,14 @@ public async Task<IActionResult> OnPost(CaseType? type, string? reason, string?
 
         reason ??= string.Empty;
         modNotes ??= string.Empty;
-        
+
         // if id is invalid then return bad request
         if (!await type.Value.IsIdValid((int)affectedId, this.Database)) return this.BadRequest();
 
-        UserEntity? affectedUserEntity =
+        UserEntity? affectedUser =
             await this.Database.Users.FirstOrDefaultAsync(u => u.UserId == affectedId.Value);
 
-        if (affectedUserEntity?.IsModerator ?? false)
+        if (affectedUser?.IsModerator ?? false)
```

Comment:
This should probably check if the affected user is null since you shouldn't be able to create a case for a nonexistent user.